lapce-data/src/data.rs

```diff
@@ -275,6 +320,135 @@ impl LapceData {
+                                    // Interpret [all components to the left of rhs2] combined as path
+                                    let path_str:String = splits.rev().map(|s| s.to_string()).collect::<String>();
+                                    let path_buf = PathBuf::from(path_str);
+                                    let path = path_buf.as_path();
+                                    
+                                    // rhs1 must be column, rhs2 must be line, and everything else to the left combined must be file name
+                                    if path.is_file() {
+                                        return Some((
+                                            path.to_path_buf(),
+                                            Some( LineCol { line: second_rhs_num, column: first_rhs_num } ),
+                                        ))
+                                    }
+                                },
+                                // Second right component isn't a number
+                                Err(_) => {
+                                    // Interpret [all components to the left of rhs1] combined as path
+                                    let mut path_str:String = splits.rev().map(|s| s.to_string()).collect::<String>();
+                                    path_str.push(':');
+                                    path_str.push_str(second_rhs);
+                                    let path_buf = PathBuf::from(path_str);
+                                    let path = path_buf.as_path();
+                                    
+                                    // rhs1 must be line number and previous components combined must be file name
+                                    if path.is_file() {
+                                        return Some((
+                                            path.to_path_buf(),
+                                            Some( LineCol { line: first_rhs_num, column: 1 } ),
+                                        ))
+                                    }
+                                },
+                            }
+                        }
+                    },
+                    // First right component isn't a number
+                    Err(_) => {
+                        // If the first right component isn't a number, then the enire path has to refer to a file name
+                        if path.is_file() {
+                            return Some((
+                                path.to_path_buf(),
+                                Some( LineCol { line: 1, column: 1 } ),
+                        ));
+                        }
+                    }
+                }
+            }
+        // Check if the path refers to a file if the path isn't UTF-8 valid
+        } else if path.is_file() {
+            return Some((
+                path.to_path_buf(), 
+                Some( LineCol { line: 1, column: 1 } )
```

Comment:
Should be
```rust
LineCol { line: second_rhs_num, column: first_rhs_num }
```

Reply:
Need to clarify the arguments for `LapceUICommand::JumpToLineColLocation` and `EditorLocation`. If with `position = None` and `scroll_offset = None` then it is automatically accepted that `position = 1, scroll_offset = (1, 1)`, then it is better return `None`